lib/matplotlib/colorbar.py

```diff
@@ -353,10 +353,9 @@ class ColorbarBase:
     ax : `~matplotlib.axes.Axes`
         The `~.axes.Axes` instance in which the colorbar is drawn.
     lines : list
-        A list of `.LineCollection` if lines were drawn, otherwise
-        an empty list.
+        A list of `.LineCollection` (empty if no lines were drawn).
     dividers : `.LineCollection`
-        A LineCollection if *drawedges* is ``True``, otherwise ``None``.
+        A LineCollection (empty if *drawedges* is ``False``).
```

Comment:
Should this get an API change note?

Reply:
I don't think this is an API break because previously the attribute could be LineCollection or None, and now it's always LineCollection, so we don't add more types to handle.

Reply:
Technically, people could have checked `cb.dividers is None`, which now would have to be changed to `not cb.dividers`.

Reply:
sure, added changelog